### PR TITLE
Ensure that multiset calls still work and warning

### PR DESCRIPTION
**Pull request by junos**
Jira Ticket:
https://quandl.atlassian.net/browse/AP-1658

## Description:
Ensure that multiset calls still work in new package

## Note to QA:
(Risk of breaking exisiting code, how to test the new code change)

### CODE REVIEW

File: quandl/util.py [hunk 1]
```diff
@@ -106,3 +106,19 @@ def convert_column_names(meta):
             return list([Util.methodize(x) for x in meta['columns']])
         else:
             return []
+
+    @staticmethod
+    def is_multiset_calls(self, dataset):
```
Comment (ccleung) [inline quandl/util.py]: why are we passing in self to a static method?

File: quandl/get.py [hunk 2]
```diff
@@ -91,3 +94,8 @@ def _convert_params_to_v3(params):
             # update to the new query param if not specified already
             if v not in params:
                 params[v] = params.pop(k)
+
+
+def _multiset_Deprecation_warning(dataset):
+    if Util.is_multiset_calls(Util, dataset):
```
Comment (ccleung) [inline quandl/get.py]: why are we passing in `Util`? i don't think that is needed

File: quandl/get.py [hunk 3]
```diff
@@ -91,3 +94,8 @@ def _convert_params_to_v3(params):
             # update to the new query param if not specified already
             if v not in params:
                 params[v] = params.pop(k)
+
+
+def _multiset_Deprecation_warning(dataset):
```
Comment (ccleung) [inline quandl/get.py]: please follow the proper naming convention, lower case everything?

**Comment by ccleung** [comment]
im actually not sure why this is needed

**Comment by junos** [comment]
confirmed with chris, this is not needed.

**Pull request closed without merging**